extension-config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate/ext-terms-and-conditions",
```

Comment:
Id should be: @shopgate/terms-and-conditions

Reply:
Updated